tests/test_config.py

```diff
@@ -296,5 +345,24 @@ def test_get_hdfs_to_snowflake_query_creates_query(self):
             """.strip())
 
 
+class TableSchemaTests(ConfigTests):
+    def test_get_table_schema_sql_no_schema(self):
+        sql = get_table_schema_sql('""."B"')
```

Comment:
It turns out that this isn't how the code behaves if no schema is specified... :-(

https://github.com/outrauk/dataiku-plugin-snowflake-hdfs/issues/19

Reply:
This is another good example that we need to test plugin on Dev Dataiku instance before committing code here.